.env.test

```diff
@@ -1,6 +1,10 @@
 KERNEL_CLASS='StepupReadId\Kernel'
 APP_ENV=dev
 APP_SECRET=secret
-APP_AUTHORIZATION_TOKEN=authorization_token
+
+APP_SUBMITTER_TOKEN=submitter_token
+APP_VIEWER_TOKEN=viewer_token
 APP_OPAQUE_ID=opaque_id
 APP_READID_SERVER_FQDN=readid.server.com
+APP_WEBHOOK_TOKEN=webhook_token
+APP_ATTRIBUTES_MAPPING='{}'
```

Comment:
I think adding instructions regarding these environment variables should end up in the README.md. You could elaborate on how to obtain the tokens.

Feel free to do this at a later stage.